lml/src/main/java/com/github/czyzby/lml/parser/impl/attribute/OnDoubleClickLmlAttribute.java

```diff
@@ -0,0 +1,73 @@
+package com.github.czyzby.lml.parser.impl.attribute;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.github.czyzby.lml.parser.LmlParser;
+import com.github.czyzby.lml.parser.action.ActorConsumer;
+import com.github.czyzby.lml.parser.tag.LmlAttribute;
+import com.github.czyzby.lml.parser.tag.LmlTag;
+
+/** Attaches a ClickListener to the the actor, invoking a chosen action upon left button double clicking on actor. Expects an action ID.
+ * Assigned action method might have {@link OnDoubleClickLmlAttribute.Params} as parameter to retrieve some event related data.
+ * Mapped to "onDoubleClick" and "doubleClick" attribute names.
+ *
+ * @author Metaphore */
+public class OnDoubleClickLmlAttribute implements LmlAttribute<Actor> {
+    private static final long SECOND_CLICK_TIME = 250000000L; // 250ms
+
+    @Override
+    public Class<Actor> getHandledType() {
+        return Actor.class;
+    }
+
+    @Override
+    public void process(final LmlParser parser, final LmlTag tag, final Actor actor, final String rawAttributeData) {
+        final ActorConsumer<?, Params> action = parser.parseAction(rawAttributeData, tmpParams);
+        if (action == null) {
+            parser.throwError("Could not find action for: " + rawAttributeData + " with actor: " + actor);
+        }
+        actor.addListener(new ClickListener(0) {
+            private boolean firstClickCaught = false;
+            private long lastClickTime = 0;
+
+            @Override
+            public void clicked(final InputEvent event, final float x, final float y) {
+                long currentEventTime = Gdx.input.getCurrentEventTime();
+                long deltaTime = currentEventTime - lastClickTime;
+                lastClickTime = currentEventTime;
+
+                if (!firstClickCaught) {
+                    firstClickCaught = true;
+                } else {
+                    if (deltaTime < SECOND_CLICK_TIME) {
+                        firstClickCaught = false;
+
+                        tmpParams.actor = actor;
+                        tmpParams.x = x;
+                        tmpParams.y = y;
+                        tmpParams.stageX = event.getStageX();
+                        tmpParams.stageY = event.getStageY();
+                        action.consume(tmpParams);
+                        tmpParams.reset();
+                    }
```

Comment:
I don't see `firstClickCaught` being cleared.

Reply:
OK, I think I know why you don't always reset the marker (it might be the beginning of another double-click). It seems OK.

Reply:
This may look weird, but it actually works right. The reason `firstClickCaught` is set to `false` only after double click is handled is to prevent to handle occasional triple click. If `firstClickCaught` gets cleared in this manner:
```
if (!firstClickCaught) {
    firstClickCaught = true;
} else {
    firstClickCaught = false;
    // ...
}
```
The code want be able to handle double click in this case:
1. Click
2. 1 second pause
3. Click (here program will register long delay and reset the flag)
4. Click